app/src/main/res/values/donottranslate_questionnaire_values.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Got it. So, in case `questionnaire_values_age` is `translatable="false"` then

res/values/strings.xml

```xml
<string name="questionaire_item_10_age" translatable="false">10代</string>
```